ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -651,7 +651,7 @@ open class SegmentedControl: UIControl {
         }
     }
 
-    private func updateWindowSpecificColors() {
+    public func updateWindowSpecificColors() {
```

Comment:
It doesn't seem like the client code would need to have access to this in any scenario other than the ColorDemoController. Is there a better way to trigger this update without making this func public?

Reply:
Calling segmentedControl.didMoveToWindow() works too, but Harrie thought it'd be better to call updateWindowSpecificColors

Reply:
I believe the price of making that a public API is higher. @harrieshin are you okay with going with the approach of calling didMoveToWindow()?

Reply:
in this case, i don't think calling didMoveToWindow() is the right thing. (1) it is something that iOS usually calls the view not client code calling client code like invalidatelayout(). (2) in reality, it didn't move the window. we are relying on bilateral effect.

Reply:
Yes. I agree that it doesn't seem like the right thing to do. When I asked this question one of the ideas I had in mind were more along the lines of setNeedsLayout.

Reply:
After looking at the code, didMoveToWindow() works because it's overridden in SegmentedControl.swift and it's calling the updateWindowSpecificColors() method.  I tried layoutSubviews, setNeedsLayout, setNeedsDisplay, and reloadInputViews before, but none of those work unfortunately.